doc/src/manual/noteworthy-differences.md

```diff
@@ -285,6 +285,9 @@ For users coming to Julia from R, these are some noteworthy differences:
     literals are rounded (and not promoted to the `BigFloat` type) if they can not be exactly represented.
      Floating point literals are closer in behavior to C/C++. Octal (prefixed with `0o`) and binary
     (prefixed with `0b`) literals are also treated as unsigned (or `BigInt` for more than 128 bits).
+  * In Julia, the division operator [`/`](@ref) always returns a floating point number.  To perform
+    integer division, use [`div`](@ref) or [`÷`](@ref div). Indexing an array with an expression
+    involving `/` as in `a[i/2]`, will cause an error in Julia.
```

Comment:
Maybe it doesn't matter but if you want to be more precise it could be good to:

- emphasize that `/` always returns a floating point number *when the two operands are of integer type*.
- change "array" to "`Array`" 
- mentiond that `i` is an `Int` in `a[i/2]`

Reply:
Updates after comments from @KristofferC (thanks!)

Reply:
÷ works as a substitute in general, 1÷2 == 0 out-of bounds, so maybe add some wording for indexing, since 1-based?